gammapy/datasets/simulate.py

```diff
@@ -21,6 +22,9 @@
 
 __all__ = ["MapDatasetEventSampler", "ObservationEventSampler"]
 
+logging.basicConfig(level=logging.INFO)
```

Comment:
Please remove the configuration statement here. This usually happens on a global call in the user script.

Reply:
If I remove it, there is no more output from `logger.info`. I converted all the `logger.info` to simple `print`.

Reply:
No, please keep the logging statements. We do not use print statements anywhere in the Gammapy code. My points was that the configuration should happen in the user scripts, or in you case, move the logger configuration to the test file.